x/bank/keeper/keeper.go

```diff
@@ -67,7 +67,9 @@ func (k BaseKeeper) GetPaginatedTotalSupply(ctx sdk.Context, pagination *query.P
 		if err != nil {
 			return fmt.Errorf("unable to convert amount string to Int %v", err)
 		}
-		supply = append(supply, sdk.NewCoin(string(key), amount))
+
+		// `Add` omits the 0 coins addition to the `supply`.
+		supply = supply.Add(sdk.NewCoin(string(key), amount))
```

Comment:
q: isn't the key prefixed? are there tests for this?

Reply:
Not necessary, here `supply` is just a local variable. Here are the tests for supply.